tests/system/providers/google/cloud/bigquery/example_bigquery_transfer.py

```diff
@@ -38,7 +38,7 @@
 
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
```

Comment:
I think you want to change this line.
```suggestion
ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID", "default")
```
you have more places where you miss to add default

Reply:
Yes, indeed, i fixed some of them, but it is actually good to have but not required for the test execution :D 
Let me add more (there are A LOT of tests without it)